index.js

```diff
@@ -72,8 +72,8 @@ app.get("/api/reviews/:id", async (req, res) => {
   }
 });
 
-app.get("/api/reviews", async (req, res) => {});
+app.get('/api/reviews', async (req, res) => {});
 
-app.listen(port, () => {
+module.exports = app.listen(port, () => {
```

Comment:
I had to export the server - otherwise in my tests I was getting an error about `app.address` not existing. There are other ways to solve this but this seemed appropriate for a small application that isn't used often - for a larger application I would want a better solution that doesn't require creating a listener during tests.